mwm-3-kotlin-1/presentation/PITCHME.md

```diff
@@ -546,6 +546,25 @@ public final void testForWithIndices() {
 
 ---
 
+### <span style="color: #00B8D4; text-transform: none; font-size:0.8em;">Extend</span> <span style="text-transform: none; font-size:0.8em;">from a parent</span>
+
+In Kotlin, class and methods are `final` by default.
+
+Use `open` keyword in from of a class/method to be able to extend from it :
+
+```kotlin
+open class Animal(){
+	open fun myOpenedFunction() {}
+}
+class Dog() : Animal(){
+	override fun myOpenedFunction() {}
+}
+```
+
+We must use `override` language keyword to override a method.
```

Comment:
Smaller sentence: `override language keyword required to override a method`

Reply:
Fixed !